### PR TITLE
Webconfig notification for all http codes

### PR DESCRIPTION
**Pull request by sadhyama**

### CODE REVIEW

File: source/WebConfig/webconfig_internal.c [hunk 1]
```diff
@@ -420,13 +412,15 @@ int handleHttpResponse(long response_code, char *webConfigData, int retry_count,
 	else if(response_code == 204)
 	{
 		WebConfigLog("No configuration available for this device. response_code:%d\n", response_code);
+		addWebConfigNotifyMsg(configURL, response_code, NULL, 0, RequestTimeStamp , "v1.0-NONE");
```
Comment (shilpa24balaji) [inline source/WebConfig/webconfig_internal.c]: Remove v1.0, its only NONE.
Schema version is separate.

File: source/WebConfig/webconfig_internal.c [hunk 2]
```diff
@@ -437,11 +431,17 @@ int handleHttpResponse(long response_code, char *webConfigData, int retry_count,
 	if((response_code !=403) && (first_digit == 4)) //4xx
 	{
 		WebConfigLog("Action not supported. response_code:%d\n", response_code);
+		addWebConfigNotifyMsg(configURL, response_code, NULL, 0, RequestTimeStamp , "v1.0-NONE");
```
Comment (shilpa24balaji) [inline source/WebConfig/webconfig_internal.c]: How come all cases its "NONE", get of old version should be used.

File: source/WebConfig/webconfig_internal.c (hunk 1 above)
Comment (sadhyama) [reply source/WebConfig/webconfig_internal.c]: Incorporated.

File: source/WebConfig/webconfig_internal.c (hunk 2 above)
Comment (sadhyama) [reply source/WebConfig/webconfig_internal.c]: Incorporated.

**Pull request merged**